css/bulma.css

```diff
@@ -3136,6 +3143,7 @@ a.tag:hover {
 
 .subtitle {
   color: #4a4a4a;
+  font-family: inherit;
```

Comment:
Some variables were displayed conditionally which doesn't work great with css variables I made the default become inherit as I though that would make more sense but if it's a problem it is possible to invalidate the css property by giving it a wrong value like `-`